source/_components/weather.ipma.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "IPMA"
+description: "Instructions on how to integrate Instituto Português do Mar e Atmosfera weather conditions into Home Assistant."
+date: 2018-05-31 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bom.png
+ha_category: Weather
+ha_release: 0.71
+ha_iot_class: "Cloud Polling"
+---
+
+The `ipma` weather platform uses the [Instituto Português do Mar e Atmosfera](http://www.ipma.pt) as a source for current and forecast meteorological data.
+
+## {% linkable_title Configuration %}
+
+To add the IPMA weather platform to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+weather:
+  - platform: ipma 
+```
+
+{% configuration %}
+name:
+  description:  The name you would like to give to the weather station.
+  required: optional
+  default: The name of the used station
+  type: string
+latitude:
+  description: "Latitude of the location for which you want weather information"
```

Comment:
This sentence is surrounded with quotes, but other sentences are not. Also missing period.